.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6.1
```

Comment:
Any reason not to use the most up to date version here (3.6.5)? [py-EVM](https://github.com/ethereum/py-evm/blob/master/.circleci/config.yml#L52) specifies just `circleci/python:3.6` which presumably chooses the most recent one available.

Reply:
Ah ... because that comes from the template that circle ci provides. `circleci/python:3.6` sounds good, will fix to this